Schemas.sql

```diff
@@ -24,28 +24,25 @@ ALTER TABLE IF EXISTS public.animals
     ALTER TABLE animals ADD COLUMN species VARCHAR(100);
 
 /* Vet clinic database: query multiple tables */
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you reset your database again in your operating system and run your queries and attach the fresh screenshots in your pull request summary because in your summary some screenshots are not giving good results but i have tested the data on my side and your queries are returning good results :+1:

Reply:
@HAMMAS-SALEEM 
Thanks for your valuable and good feedback.
I will consider your nice suggestion.